dlrs/main/objects/DeclarativeLookupRollupSummaries__c/fields/DisableDLRSGlobally__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>DisableDLRSGlobally__c</fullName>
```

Comment:
@sharicarlson13 FullName, description, help text, and label review, please :)

Reply:
Looks good to me